backend/start_api.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash 
+
+# check for uvicorn workers env var, default to 1
+if [ -z "${UVICORN_WORKERS}" ]; then
+    UVICORN_WORKERS=1
+fi
```

Comment:
This works as-is, but there's a shorthand version in bash for setting a variable to a default if it doesn't exist:

```suggestion
UVICORN_WORKERS=${UVICORN_WORKERS:-1}
```

The suggestion attempts to use `${UVICORN_WORKERS}` from the environment first, then if it's empty or unset uses the default value `1`.